proto/slinky/marketmap/v1/market.proto

```diff
@@ -0,0 +1,109 @@
+syntax = "proto3";
+package slinky.marketmap.v1;
+
+option go_package = "github.com/skip-mev/slinky/x/marketmap/types";
+
+import "gogoproto/gogo.proto";
+
+// Ticker represents a price feed for a given asset pair i.e. BTC/USD. The price feed
+// is scaled to a number of decimal places and has a minimum number of providers required
+// to consider the ticker valid.
+message Ticker {
+    // ID is a unique sequentially-generated value for the ticker.
+    uint64 id = 1;
```

Comment:
this should have a 1:1 mapping with what is stored in the `x/oracle` module.

Reply:
Do we actually need this?  Since the kvstore access in `x/oracle` is `BASE/QUOTE` doesnt using that already give use 1:1 mapping?